scripts/ec2cleanup.py

```diff
@@ -189,8 +192,34 @@ def delete_snapshots(provider_mgmt, excluded_snapshots, output):
         logger.exception('Exception in %r', delete_snapshots.__name__)
 
 
+def delete_queues(provider_mgmt, excluded_queues, output):
+    queue_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    current_timestamp = time.time()
+    try:
+        queues = provider_mgmt.list_queues_with_creation_timestamps()
+        for queue in queues:
+            if excluded_queues and queue in excluded_queues:
+                logger.info(" Excluding Queue with url: %r", queue)
+                continue
+            else:
+                # 1209600 is 14 days in seconds
+                if float(current_timestamp) - float(queues[queue]) > 1209600:
+                    provider_mgmt.delete_sqs_queue(queue)
+                    queue_list.append([provider_name, queue])
+        logger.info("  Deleted Queues: %r", queue_list)
+        with open(output, 'a+') as report:
+            if queue_list:
```

Comment:
Optional:
I would open a file for writing only if `queue_list` available.

Reply:
You are right. That makes sense.